content/en/opentelemetry/interoperability/_index.md

```diff
@@ -18,10 +18,11 @@ Integrating Datadog with OpenTelemetry allows you to use Datadog's comprehensive
     {{< /nextlink >}}
     {{< nextlink href="/tracing/trace_collection/trace_context_propagation/" >}}W3C Trace Context Propagation{{< /nextlink >}}
     {{< nextlink href="/opentelemetry/interoperability/api_support" >}}OpenTelemetry API Support{{< /nextlink >}}
+    {{< nextlink href="/opentelemetry/interoperability/drop_in_support" >}}Using OpenTelemetry Instrumentations{{< /nextlink >}}
```

Comment:
What if we rename the shortname `drop_in_support` and the long name `Using OpenTelemetry Instrumentations` to `OpenTelemetry Instrumentation Support`. Thoughts @mcculls @tushar-datadog ?

Reply:
The terms are a little hand-wavey, but it seems like the term may be `Instrumentation Library` for the most up-to-date OpenTelemetry docs: https://opentelemetry.io/docs/specs/otel/overview/#instrumentation-libraries

Reply:
`Instrumentation Library` is only part of the story for Java - that defines library code that interacts with the OTel APIs (whether that is code shipped with the library, or available separately using library hooks to call into OTel.)

Most Java instrumentation is what they call [zero-code](https://opentelemetry.io/docs/concepts/instrumentation/zero-code/) or `agent-based` which involves bytecode manipulation.

Both `agent-based` instrumentation and `Instrumentation Libraries` fall under `automatic instrumentation`.

Reply:
Going to go with `Instrumentation Libraries` because it's less vague than the other options - it's not quite the full story for Java, but we can cover that in the Java specific text